package.json

```diff
@@ -22,19 +25,21 @@
   },
   "homepage": "https://github.com/ProjectProvenance/ProofPoints#readme",
   "dependencies": {
+    "bignumber.js": "^9.0.0",
```

Comment:
What's this for?

Reply:
Its required by the package `web3-eth` but not mentioned as an explicit dependency of that package, which seems to be a bug in that package, but this is the workaround.